### PR TITLE
test.py counts lines in an input text file

### PR DESCRIPTION
**Pull request by aliz237**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,6 +2,7 @@
 __pycache__/
 *.py[cod]
 *$py.class
+*~
```
Comment (aliz237) [inline .gitignore:5]: good idea

Comment (aliz-umd) [reply .gitignore:5]: simplify gitignore

**Review by aliz-umd — CHANGES_REQUESTED** [review]
make the necessary changes

File: .gitignore [hunk 2]
```diff
@@ -2,6 +2,7 @@
 __pycache__/
 *.py[cod]
```
Comment (aliz-umd) [inline .gitignore:3]: what's this line for?

File: test.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import sys
+
+def f(path):
+    with open(path) as fin:
+        lines = fin.readlines()
+    print(f'file {path} has {len(lines)} lines.')
+
+if __name__ == '__main__':
+    f(sys.argv[1])
```
Comment (aliz-umd) [inline test.py:9]: yes use argparse

File: textfile [hunk 4]
```diff
@@ -0,0 +1,6 @@
+line1
+line2
+line3
+
+line5
+line6
```
Comment (aliz-umd) [inline textfile:6]: fix this, all else looks good.

**Review by aliz-umd — CHANGES_REQUESTED** [review]

**Pull request merged**